bunch-cli/src/main/kotlin/org/jetbrains/bunches/git/ignore.kt

```diff
@@ -16,6 +19,24 @@ private fun isGradleBuildDir(dir: File): Boolean {
     return false
 }
 
+fun parseGitIgnore(baseGitRoot: File): IgnoreNode? {
+    val gitignoreFile = File(baseGitRoot, ".gitignore")
+    try {
+        FileInputStream(gitignoreFile).use {
```

Comment:
Kotlin has `inputStream()`, can it be used here?

Reply:
Yes, fixed in c6b1b17